editmsg.c

```diff
@@ -118,7 +121,19 @@ static int edit_one_message(struct Context *ctx, struct Header *cur)
     goto bail;
   }
 
-  mtime = mutt_decrease_mtime(tmp, &sb);
+  /* remove write permissions */
+  if (!edit)
+  {
+    rc = chmod(tmp, sb.st_mode & ~(S_IWUSR | S_IWGRP | S_IWOTH));
+    if (rc == -1)
+    {
+      mutt_error(_("Could not remove write permissions of %s: %s"), tmp, strerror(errno));
+      goto bail;
```

Comment:
Yep, sorry for not getting to this earlier. I'd probably not bail out, just log something and let it be :)

Reply:
Will do it shortly.